enterprise/reporting/label_set.go

```diff
@@ -32,6 +32,8 @@ type LabelSet struct {
 	EventName               string
 	EventType               string
 	ErrorType               string
+	ErrorCode               string
+	ErrorMessage            string
```

Comment:
@koladilip Many columns which are present for metrics like source_category, `source_job_id`, `source_job_run_id` etc are not there for errors reporting. So we are going to rely on the default values of these columns for error reporting ?

Reply:
yes, for this PR this is fine, we can take a look at these later if they are. needed or not